06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(file):
+    data = {}
+    print
+    with open(file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                if name not in data:
+                    data[name] = score
+                else:
+                    if score > data[name]:
+                        data[name] = score
+
+        except ValueError:
+            print("Could not parse csv")
```

Comment:
If an exception occurred you print an error message and continue with the program, which would eventually return the data object (empty or half empty). I think that in case of an error you should return None, or something similar, that way the main function will know that something went wrong.